### ISSUE TITLE
kube-api-qps and kube-api-burst flags not available in updater component

### ISSUE THREAD

**Issue by adriananeci** [kubernetes/autoscaler#4535]
**Which component are you using?**:
vertical-pod-autoscaler

**What version of the component are you using?**:
0.9.2

Component version:

**What k8s version are you using (`kubectl version`)?**:

<details><summary><code>kubectl version</code> Output</summary><br><pre>
❯ kubectl version
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.0", GitCommit:"ab69524f795c42094a6630298ff53f3c3ebab7f4", GitTreeState:"clean", BuildDate:"2021-12-07T18:08:39Z", GoVersion:"go1.17.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.11", GitCommit:"27522a29febbcc4badac257763044d0d90c11abd", GitTreeState:"clean", BuildDate:"2021-09-15T19:16:25Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/amd64"}

</pre></details>

**What environment is this in?**:

**What did you expect to happen?**:
Below `updater` flags are present on `master` branch, but are missing from `vpa-release-0.9` branch. (https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/updater/main.go)

```
	kubeApiQps   = flag.Float64("kube-api-qps", 5.0, `QPS limit when making requests to Kubernetes apiserver`)
	kubeApiBurst = flag.Float64("kube-api-burst", 10.0, `QPS burst limit when making requests to Kubernetes apiserver`)
```

Make QPS limit and burst available for updater too in order to amke it configurable and avoid request throttling like:
```
I1217 15:28:47.051676       1 request.go:557] Throttling request took 381.003307ms, request: POST:https://192.168.0.1:443/api/v1/namespaces/kube-system/pods/cluster-autoscaler-dc76d98c4-bvv84/eviction


I1217 15:28:48.051716       1 request.go:557] Throttling request took 394.457428ms, request: POST:https://192.168.0.1:443/api/v1/namespaces/monitoring/events
```

**What happened instead?**:

```
unknown flag: --kube-api-qps
Usage of /updater:
unknown flag: --kube-api-qps
      --add-dir-header                       If true, adds the file directory to the header
      --address string                       The address to expose Prometheus metrics. (default ":8943")
      --alsologtostderr                      log to standard error as well as files
      --evict-after-oom-threshold duration   Evict pod that has only one container and it OOMed in less than
                                             		evict-after-oom-threshold since start. (default 10m0s)
      --eviction-rate-burst int              Burst of pods that can be evicted. (default 1)
      --eviction-rate-limit float            Number of pods that can be evicted per seconds. A rate limit set to 0 or -1 will disable
                                             		the rate limiter. (default -1)
      --eviction-tolerance float             Fraction of replica count that can be evicted for update, if more than one pod can be evicted. (default 0.5)
      --log-backtrace-at traceLocation       when logging hits line file:N, emit a stack trace (default :0)
      --log-dir string                       If non-empty, write log files in this directory
      --log-file string                      If non-empty, use this log file
      --log-file-max-size uint               Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
      --logtostderr                          log to standard error instead of files (default true)
      --min-replicas int                     Minimum number of replicas to perform update (default 2)
      --skip-headers                         If true, avoid header prefixes in the log messages
      --skip-log-headers                     If true, avoid headers when opening log files
      --stderrthreshold severity             logs at or above this threshold go to stderr
      --updater-interval duration            How often updater should run (default 1m0s)
      --use-admission-controller-status      If true, updater will only evict pods when admission controller status is valid. (default true)
  -v, --v Level                              number for the log level verbosity (default 0)
      --vmodule moduleSpec                   comma-separated list of pattern=N settings for file-filtered logging
      --vpa-object-namespace string          Namespace to search for VPA objects. Empty means all namespaces will be used.
```

**How to reproduce it (as minimally and precisely as possible)**:
Try to configure `--kube-api-qps` or `kube-api-burst` for `0.9.2` updater component 

**Anything else we need to know?**:

**Comment by jbartosik** [comment]
I'm planning a new release (#4404), which will fix this.

**Comment by jbartosik** [comment]
I released [0.10.0](https://github.com/kubernetes/autoscaler/releases/tag/vertical-pod-autoscaler-0.10.0) on 2022-01-26, I'm closing this issue.

Please reopen if this doesn't solve your problem.
